src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Shape shape = Shape.values()[random.nextInt(Shape.values().length)];
+
+        switch (shape) {
+            case CIRCLE:
+                return new Circle(random.nextInt(10) + 1,
+                                        colorSupplier.gerRandomColor());
+            case SQUARE:
+                return new Square(random.nextInt(10) + 1,
```

Comment:
It would be better if you make methods for returning figures and use it instead of creating objects via new directly